### PR TITLE
ABySS foss2016a

### PR DESCRIPTION
**Pull request by RvDijk**
ABySS for FOSS 2016a

### CODE REVIEW

**Comment by boegel** [comment]
@RvDijk why did you close this again?

**Comment by RvDijk** [comment]
@boegel the dependency for sparsehash in the configuration file did contain syntax errors, it is now solved.

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/a/ABySS/ABySS-1.9.0-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['https://github.com/bcgsc/abyss/releases/download/%(version)s/']
+
+dependencies = [
+	('Boost', '1.60.0'),
+	('sparsehash', '2.0.2')
```
Comment (boegel) [inline easybuild/easyconfigs/a/ABySS/ABySS-1.9.0-foss-2016a.eb]: please avoid using tabs for indentation, use spaces instead (one indent level == 4 spaces)

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2157.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/4c58380810271a486a0834898a2c3543 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2692.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/5896f66bc8c67e81b919985c5d8f6a4b for a full test report.

**Comment by boegel** [comment]
@RvDijk missing dependency on SQLite?

```
error: cannot find -lsqlite3
collect2: error: ld returned 1 exit status
```

**Comment by RvDijk** [comment]
@boegel dependency for SQLite is added.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2718.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6905fc32c0bab8da55efded0e75ca5da for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2017.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/aa586a1830dc84a8f323ae36d6fafd5f for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @RvDijk!

**Pull request merged**